### ISSUE TITLE
Please increase TypeId limit to at least 2 bytes!

### ISSUE THREAD

**Issue by cbalmeida** [hivedb/hive#735]
Hive is such a good thing to flutter projects, but with this limit of 123 typeIds, we just can't use it in replacement of an SQLite database with more than 1,000 tables!

Is this so hard to do?

**Comment by TheMisir** [comment]
> Is this so hard to do?

No, but doing so will break all the exists implementations. We might implement a feature to set project wide typescpace size so the previous implementations won't break while new implementations could write more than 123 types.
